cmd/tealdbg/localLedger.go

```diff
@@ -130,6 +162,28 @@ func makeAppLedger(
 	return ledger, states, err
 }
 
+func getBalanceFromIndexer(indexerURL string, account basics.Address, round int) (basics.AccountData, error){
+	queryString := fmt.Sprintf("%s/v2/accounts/%s?round=%d", indexerURL, account, round)
+	resp, err := http.Get(queryString)
+	if err != nil {
+		return basics.AccountData{}, err
+	}
+	if resp.StatusCode != 200 {
+		return basics.AccountData{}, fmt.Errorf("indexer error response status code %d", resp.StatusCode)
```

Comment:
If StatusCode is, let's say, 400, the resp.Body would not get closed..
The "classic" go design patten here is to call
`defer resp.Body.Close()` right after checking the error code from "Get".

Reply:
Plus, we want to see error text from the indexer if any

Reply:
Error text meaning the response right? If so, what would be the best way to fetch that? Would Ioutil.ReadAll() be a good method? This might not be the best approach if the endpoint returned a high amount of data?

Reply:
`Ioutil.ReadAll()` looks OK.